annotations/1.3/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Jakarta Annotations 1.3"
+date: 2019-08-01T11:00:00+0100
+summary: "Draft for Jakarta EE 8"
+---
+Jakarta Annotations defines a collection of annotations representing common semantic concepts that
+enable a declarative style of programming that applies across a variety of Java technologies.
+
+* [Jakarta Annotations 1.3 Specification Document](./annotations-spec-1.3.pdf) (PDF)
+* [Jakarta Annotations 1.3 Specification Document](./annotations-spec-1.3.html) (HTML)
+* [Jakarta Annotations 1.3 Javadoc](./apidocs)
+* [Jakarta Annotations 1.3 TCK](http://download.eclipse.org/ee4j/jakartaee-tck/jakartaee8-eftl/promoted/eclipse-annotations-tck-1.3.0.zip)
```

Comment:
Should be of form:
https://download.eclipse.org/jakartaee/{spec}/x.y/*{spec}-tck-x.y.z.zip

Reply:
Fixed